terraform/aws/scenarios/omnibus-external-openldap/files/chef-server.rb

```diff
@@ -8,10 +8,10 @@
 
 profiles['root_url'] = 'http://localhost:9998' unless profiles.nil?
 
-ldap['base_dn'] = 'ou=chefs,dc=chef-server,dc=dev'
```

Comment:
per the meetup, we dont need any of the terraform parts updated, this is used for testing.